Update README.md to suggest how users should (#1)
ACCESS-Community-Hub/.github: profile/README.md

for the Community. Material supported directly by the ACCESS-NRI is hosted on the 
[ACCESS-NRI GitHub organisation](https://github.com/ACCESS-NRI).

ACCESS-NRI retains formal administrative control of this organisation. Access is given to ACCESS-NRI staff and community 

> members who wish to contribute where the access procedure is described in the README of each repository.

Reviewer: ```suggestion
members who wish to contribute. The procedure to request access should be described in the README of each repository.
```

Author: Can we [please resolve this question](https://github.com/ACCESS-Community-Hub/.github/pull/1#issuecomment-3181430670), before merging/closing this PR.
